whylog/converters/exceptions.py

```diff
@@ -0,0 +1,9 @@
+from whylog.exceptions import WhylogError
+
+
+class UnsupportedConverter(WhylogError):
+    def __init__(self, converter_class_name):
+        self.converter_class_name = converter_class_name
+
+    def __str__(self):
+        return 'This whylog version do not handle %s. Please upgrade Whylog' % self.converter_class_name
```

Comment:
I suggest left this fragment. This error could by caused, because some use in parser new Converted which is not supported in installed whylog version. So Whylog upgarde could be helpful